src/app/process-editor/store/process-editor.effects.ts

```diff
@@ -247,7 +248,11 @@ export class ProcessEditorEffects extends BaseEffects {
                 logInfo(getProcessLogInitiator(), this.translation.instant('PROCESS_EDITOR.PROCESS_UPDATED')),
                 new SnackbarInfoAction('PROCESS_EDITOR.PROCESS_UPDATED')
             ]),
-            catchError(e => this.genericErrorHandler(this.handleProcessUpdatingError.bind(this), e))
+            catchError(e => {
+                this.store.dispatch(new UpdateProcessFailedAction());
```

Comment:
I don't think we need to directly dispatch this action.
Can't we just return this action as part of the handleProcessUpdatingError, where we are returning the SnackbarErrorAction as well?